openapi/definitions.yaml

```diff
@@ -247,6 +247,30 @@ CreatedMessageWithoutContent:
     - fiscal_code
     - created_at
     - sender_service_id
+EnrichedMessage:
+  type: object
+  properties:
+    id:
+      type: string
+    fiscal_code:
+      $ref: "#/FiscalCode"
+    time_to_live:
+      $ref: "#/TimeToLiveSeconds"
+    created_at:
+      $ref: "#/Timestamp"
+    sender_service_id:
+      $ref: "#/ServiceId"
+    service_name:
+      type: string
+    organization_name:
+      type: string
+    message_title:
+      type: string
```

Comment:
I'm thinking about these attributes. Because of retro-compatibility maybe we should keep them optional, but over the last API version these values are always present AFAIK. What do you think about?

Reply:
These attributes will be updated according to rollout. In future I think we'll have also to get rid of PaginatedEnrichedMessageCollection and related models, and maybe also EnrichedMessages, to embrace a new more clear unified model. I would keep them optional until we will get rid of the path param that says to enrich content or not.
When content will be enriched by default, we could make them required.